packages/drivers/driver-base/src/documentDeltaConnection.ts

```diff
@@ -91,6 +91,16 @@ export class DocumentDeltaConnection
     protected _disposed: boolean = false;
     private readonly mc: MonitoringContext;
 
+    /**
+     * Store op content when lightOpAcksEnabled is true
+     * This content will be used to repopulate when receiving ack for op
+     * Key: clientSequenceNumber
+     * Value: op contents
+     */
+    private readonly opContent = new Map<number, any>();
```

Comment:
content should always be a string. Typing on messages sucks, but if we can make it string everywhere else, that would be  nice. That said, in the future we will support array buffer as well, so maybe keeping it as-is is fine

Reply:
The `contents` type is currently `any` for both `IDocumentMessage` and `ISequencedDocumentMessage`